testutil/keeper/application.go

```diff
@@ -18,8 +20,15 @@ import (
 	mocks "pocket/testutil/application/mocks"
 	"pocket/x/application/keeper"
 	"pocket/x/application/types"
+	gatewaytypes "pocket/x/gateway/types"
 )
 
+var AddrToPubKeyMap map[string]cryptotypes.PubKey
+
+func init() {
+	AddrToPubKeyMap = make(map[string]cryptotypes.PubKey)
```

Comment:
Is this necessary? I think the declaration on its own achieves the same thing.

Reply:
This is necessary go doesn't like an uninitiated map iirc

Reply:
I tested and confirmed that this is correct.